build.sbt

```diff
@@ -23,21 +26,12 @@ scalacOptions ++= Seq(
   "-unchecked",
   "-Xcheckinit",
   "-Xlint:_",
-  "-Yno-adapted-args",
-  "-Ypartial-unification",
   "-Ywarn-dead-code",
-  "-Ywarn-inaccessible",
-  "-Ywarn-infer-any",
-  "-Ywarn-nullary-override",
-  "-Ywarn-nullary-unit",
   "-Ywarn-numeric-widen",
   "-Ywarn-unused",
-  "-Ywarn-unused-import",
```

Comment:
Are these options now enabled by default or something?

Reply:
I guess that `-Ywarn-unused` (that we're still using) now contains multiple options and `imports` is one of them. See https://github.com/scala/scala/pull/6632/files#diff-b36c38eab08f161687c5e199af114fb83b24a536268da451c1d009b859cfa226R114.

I checked that with the new settings, we're still getting `unused import` warnings, but other than did, I just did made minimal effort to just make this compile.